src/main/java/seedu/address/logic/commands/SortIndividualCommand.java

```diff
@@ -0,0 +1,89 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FIELD;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ORDER;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Property;
+
+/**
+ * Sorts the list of properties in the address book by price based on the order input by user.
+ */
+public class SortIndividualCommand extends Command {
+    public static final String COMMAND_WORD = "sorti";
+    private static String order;
+    private static String field;
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the list of properties based on input field.\n"
```

Comment:
Could it be clearer if you specify the that sorting is for each person?

"Sorts the list of properties for each person in the AddressBook, based on the input field\n"